lib/handlers.js

```diff
@@ -52,10 +54,16 @@ function handleFindMovieIntent(req, resp) {
 
 function handleAddMovieIntent(req, resp) {
   var movieName = req.slot('movieName');
-
-  cp.movie.search(movieName).then(function (movies) {
-    utils.sendSearchResponse(movies, resp);
-  });
+ 
+ cp.movie.search(movieName,5).then(function (movies) {
+   
+   movies = utils.formatSearchResults(movies);
+   utils.sendSearchResponse(movies, movieName, resp);
```

Comment:
That's because in sendSearchResponse it looks for movieName which was not being passed in